ngraph/core/include/openvino/core/preprocess/pre_post_process.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/core_visibility.hpp"
+#include "openvino/core/preprocess/input_info.hpp"
+
+namespace ov {
+
+class Function;
+
+namespace preprocess {
+
+class PrePostProcessorImpl;
```

Comment:
can we move this inside the `PrePostProcessor`?

Reply:
Done